app/src/main/java/org/alexcode/snake/EditNameDialog.java

```diff
@@ -51,9 +52,14 @@ public void onClick(DialogInterface dialogInterface, int i) {
                     @Override
                     public void onClick(DialogInterface dialogInterface, int i) {
                         newPlayerName = editTextPlayerName.getText().toString();
-                        playerId = PlayerPreferences.getPlayerId();
-                        if(!newPlayerName.equals(playerName)) {
-                            updatePlayerName();
+                        if(newPlayerName.length() > 15) {
```

Comment:
```suggestion
                        if (newPlayerName.length() > 15) {
```
let's be consistents

Reply:
ok